test/functional/admin/legacy_promotional_features_controller_test.rb

```diff
@@ -0,0 +1,126 @@
+require "test_helper"
+
+class Admin::LegacyPromotionalFeaturesControllerTest < ActionController::TestCase
```

Comment:
`tests  Admin::PromotionalFeaturesControllerTest` will point these tests at the correct controller.